ops/internal/config/superchain.go

```diff
@@ -10,6 +18,12 @@ const (
 	SepoliaDev0Superchain Superchain = "sepolia-dev-0"
 )
 
+var SuperchainChainIds = map[Superchain]uint64{
```

Comment:
Aren't these chain IDs already included in the superchain config? Why are we making a new hardcoded map here?

Reply:
Done in https://github.com/ethereum-optimism/superchain-registry/pull/951/commits/e7b55db0fd5d180a19bf9bba848d3d9e8d8d0f86